src/CopyrighFixer/CopyrighFixer/owner.cpp

```diff
@@ -20,12 +20,38 @@ const QString &Owner::getOwnerName() const {
     return _name;
 }
 
-void Owner::setTimeRange(const QString &interval) {
-    _timeRange = interval;
+void Owner::setTimePoint(const int &interval) {
+    _timePoint = interval;
 }
 
-const QString &Owner::getTimeRange() const {
-    return _timeRange;
+const int &Owner::getTimePoint() const {
+    return _timePoint;
+}
+
+void Owner::fromjson(const QJsonObject &objJs) {
+    if (objJs.contains("name") && objJs.value("name").isString()) {
+         _name = objJs.value("name").toString();
+    }
+
+    if (objJs.contains("timePoint")) {
+        _timePoint = objJs.value("timePoint").toInt();
+    }
+}
+
+const QJsonObject &Owner::toJson(QJsonObject &objJs) const {
```

Comment:
hmm. it is work fast.
I recommend remove the return type because your argument it is non const reference.  You can use input argument as a result argument. 
```suggestion
void Owner::toJson(QJsonObject &objJs) const {
```